imagetool/src/main/java/com/oracle/weblogic/imagetool/cachestore/FileCacheStore.java

```diff
@@ -82,15 +83,15 @@ public boolean hasMatchingKeyValue(String key, String value) {
 
     @Override
     public void addToCache(String key, String value) throws CacheStoreException {
-        Objects.requireNonNull(key, "key cannot be null");
+        Objects.requireNonNull(key, KEY_NOT_NULL);
         Objects.requireNonNull(value, "Cache item value cannot be null");
```

Comment:
Should this be coded the same as KEY_NOT_NULL ?  or Do we need internationization?

Reply:
Done